### PR TITLE
NCAR GeoCAT-Datafiles package

### PR DESCRIPTION
**Pull request by pilotchute**
…comp

Signed-off-by: pilotchute <akootz@ucar.edu>

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pilotchute** [comment]
@conda-forge/staged-recipes
Ready for your approval. :)

**Comment by pilotchute** [comment]
@conda-forge/staged-recipes NOT READY, please don't merge yet

**Comment by pilotchute** [comment]
Nevermind, --ignore-installed argument fixed it.

**Comment by pilotchute** [comment]
https://github.com/orgs/conda-forge/teams/staged-recipes
Ready for your approval. :)
The packaged size issue was an easier fix than I had anticipated.

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [inline recipes/geocat-datafiles/meta.yaml:44]: Please hardcode the name here
```suggestion
  dev_url: https://github.com/NCAR/geocat-datafiles
```

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [inline recipes/geocat-datafiles/meta.yaml:39]:
```suggestion
test:
  imports:
    - geocat.datafiles
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [inline recipes/geocat-datafiles/meta.yaml:13]:
```suggestion
```

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [inline recipes/geocat-datafiles/meta.yaml:19]:
```suggestion
```

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [inline recipes/geocat-datafiles/meta.yaml:27]:
```suggestion
```

I suggest you wait until you're in a feedstock to do this for cross-compiling; for now, it is not needed.

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [inline recipes/geocat-datafiles/meta.yaml:53]: Everyone listed should consent in a comment here ideally, unless this is like a common team where everyone goes on projects together :)

(Alternatively, you can just add yourself for now, and later in a feedstock, you can add the others)

**Review by ngam — CHANGES_REQUESTED** [review]
Looks ready, some minor edits and suggestions that shouldn't change the actual content

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [reply recipes/geocat-datafiles/meta.yaml:53]: Yes, but...

> * [x]  GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

from the main body of this PR (you incorrectly checked this item off)

Comment (dopplershift) [reply recipes/geocat-datafiles/meta.yaml:53]: We don't want to be spamming people with notifications they don't want, so every maintainer for every recipe has to comment that they're willing to be listed as a maintainer (or add a :+1: reaction).

**Comment by michaelavs** [comment]
I agree to being listed as a maintainer

**Comment by erogluorhan** [comment]
I am willing to help maintain, too!

File: recipes/geocat-datafiles/meta.yaml
Comment (anissa111) [reply recipes/geocat-datafiles/meta.yaml:53]: I am willing to be a maintainer

**Comment by hCraker** [comment]
Hi all. I'm willing to maintain this. Thanks Alea!

**Comment by anissa111** [comment]
I am willing to be a maintainer

File: recipes/geocat-datafiles/meta.yaml
Comment (pilotchute) [reply recipes/geocat-datafiles/meta.yaml:13]: These commented out lines are used by our internal pre-release testing, we want to keep the pre-release meta.yaml file as similar to the conda-forge feedstock as possible. 

May we keep them?

File: recipes/geocat-datafiles/meta.yaml
Comment (pilotchute) [reply recipes/geocat-datafiles/meta.yaml:19]: These commented out lines are used by our internal pre-release testing, we want to keep the pre-release meta.yaml file as similar to the conda-forge feedstock as possible.

May we keep them?

File: recipes/geocat-datafiles/meta.yaml
Comment (pilotchute) [reply recipes/geocat-datafiles/meta.yaml:57]: All comments are made.

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [reply recipes/geocat-datafiles/meta.yaml:27]: Sure :)

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [reply recipes/geocat-datafiles/meta.yaml:13]: Of course you can keep them. Btw, you can also start a dev channel for more regular testing — it may be handy here: https://conda-forge.org/docs/maintainer/updating_pkgs.html#maintaining-several-versions

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [reply recipes/geocat-datafiles/meta.yaml:19]: (but yes, you can keep them if you want)

File: recipes/geocat-datafiles/meta.yaml
Comment (pilotchute) [reply recipes/geocat-datafiles/meta.yaml:13]: We'll look into that. Thanks! 
For the next few weeks we are going to be focused on getting the rest of our packages into conda-forge, then we'll start updating our automation.

**Review by ngam — APPROVED** [review]
This looks good; @dopplershift could you have one final look and approve/request changes?

Some minor comments are unresolved, but the team is experienced that I wouldn't worry about them.

File: recipes/geocat-datafiles/meta.yaml
Comment (pilotchute) [reply recipes/geocat-datafiles/meta.yaml:48]: If it causes any issues at all we'll hardcode it. for now, we are maintaining this by hand, and limiting the number of changes we need to make while our packages are on both NCAR's conda channel and conda-forge, should help avoid human error.

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [reply recipes/geocat-datafiles/meta.yaml:13]: Sounds good! Thanks for all the great work :) Hopefully @dopplershift can merge soon for you

File: recipes/geocat-datafiles/meta.yaml
Comment (ngam) [reply recipes/geocat-datafiles/meta.yaml:27]: I want to add one thing for future reference: Since this is a noarch, this stuff is simply not needed. A noarch package can just be pulled into whatever arch without any modification (including pypy, arm64, etc.). But you may want to keep it for other packages which are noarch, up to you.

File: recipes/geocat-datafiles/meta.yaml
Comment (dopplershift) [inline recipes/geocat-datafiles/meta.yaml:32]: pooch is only needed at run time, not package build time:
```suggestion
  host:
    - python >=3.8
    - pip
    - setuptools
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a minor tweak to the dependencies and I'm willing to merge.

Regarding all the things kept to keep synchronized with your recipe elsewhere, I want to caution that this is going to be potentially problematic. Conda forge has bots that help maintain recipes and help migrate various changes, making changes to the recipe as necessary. You'll probably be better off using the conda-forge recipe as the canonical recipe and using it when you need to do testing.

conda-forge also has a bot that will automatically issue version updates, so that's something else to keep in mind.

**Comment by pilotchute** [comment]
@dopplershift We do intend to have conda-forge be the canonical recipe, but we will be cross maintaining packages for a month or two and want to limit complexity during that time.

File: recipes/geocat-datafiles/meta.yaml
Comment (pilotchute) [reply recipes/geocat-datafiles/meta.yaml]: ok, pooch removed.

File: recipes/geocat-datafiles/meta.yaml
Comment (pilotchute) [reply recipes/geocat-datafiles/meta.yaml]: Thanks.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**
